app/views/characters/replace.haml

```diff
@@ -16,11 +16,11 @@
 - content_for :aliases do
   - if @character.aliases.exists?
     - collection = options_from_collection_for_select(@character.aliases, :id, :name)
-    = select_tag :orig_alias, (options_for_select({ '— Any Alias —': 'all' }, 'all') + collection), prompt: '— No alias —'
+    = f.select :orig_alias, options_for_select({ '— Any Alias —': 'all' }, 'all') + collection, prompt: '— No alias —'
 
 - content_for :options do
-  = select_tag :icon_dropdown, options_for_select(@alt_dropdown, @alt.try(:id)), prompt: "No Character"
-  = select_tag :alias_dropdown, options_from_collection_for_select(@alt.try(:aliases), :id, :name), prompt: '— No alias —', class: ('hidden' unless @alt.try(:aliases).present?)
+  = f.select :icon_dropdown, options_for_select(@alt_dropdown, @alt.try(:id)), prompt: "No Character"
+  = f.select :alias_dropdown, options_from_collection_for_select(@alt.try(:aliases), :id, :name), prompt: '— No alias —', class: ('hidden' unless @alt.try(:aliases).present?)
```

Comment:
Seems to be missing `f` here - may need to pass it in in the `content_for` thing somehow? Not sure